arrow/arrow.py

```diff
@@ -407,6 +407,68 @@ def span_range(cls, frame, start, end, tz=None, limit=None, bounds="[)"):
         _range = cls.range(frame, start, end, tz, limit)
         return (r.span(frame, bounds=bounds) for r in _range)
```

Comment:
`span` has a count argument that would work here.

We could add a `count` positional argument to `span_range`, the only slight problem is that it has to go at the end else it's a breaking change, not a huge deal but slightly inconsistent.

```shell
>>> start = datetime(2013, 5, 5, 12, 30)
>>> end = datetime(2013, 5, 5, 17, 15)
>>> for r in arrow.Arrow.span_range('hour', start, end):
...     print(r)
... 
(<Arrow [2013-05-05T12:00:00+00:00]>, <Arrow [2013-05-05T12:59:59.999999+00:00]>)
(<Arrow [2013-05-05T13:00:00+00:00]>, <Arrow [2013-05-05T13:59:59.999999+00:00]>)
(<Arrow [2013-05-05T14:00:00+00:00]>, <Arrow [2013-05-05T14:59:59.999999+00:00]>)
(<Arrow [2013-05-05T15:00:00+00:00]>, <Arrow [2013-05-05T15:59:59.999999+00:00]>)
(<Arrow [2013-05-05T16:00:00+00:00]>, <Arrow [2013-05-05T16:59:59.999999+00:00]>)
(<Arrow [2013-05-05T17:00:00+00:00]>, <Arrow [2013-05-05T17:59:59.999999+00:00]>)
>>> for r in arrow.Arrow.span_range('hour', start, end, count=1.5):
...     print(r)
... 
(<Arrow [2013-05-05T12:00:00+00:00]>, <Arrow [2013-05-05T13:29:59.999999+00:00]>)
(<Arrow [2013-05-05T13:00:00+00:00]>, <Arrow [2013-05-05T14:29:59.999999+00:00]>)
(<Arrow [2013-05-05T14:00:00+00:00]>, <Arrow [2013-05-05T15:29:59.999999+00:00]>)
(<Arrow [2013-05-05T15:00:00+00:00]>, <Arrow [2013-05-05T16:29:59.999999+00:00]>)
(<Arrow [2013-05-05T16:00:00+00:00]>, <Arrow [2013-05-05T17:29:59.999999+00:00]>)
(<Arrow [2013-05-05T17:00:00+00:00]>, <Arrow [2013-05-05T18:29:59.999999+00:00]>)
```